py/mpprint.h

```diff
@@ -79,4 +79,44 @@ int mp_printf(const mp_print_t *print, const char *fmt, ...);
 int mp_vprintf(const mp_print_t *print, const char *fmt, va_list args);
 #endif
 
+// Debug messages during code developing.
+// An approximate hierarchy of debug levels is:
+// -1 - SUPPRES all messages in release version
+// 0 - the most CRITICAL errors, often requiring a system reset, use message with this level if possible raising an exception
+// 1 - ERROR requiring program restart, use message with this level before raising an exception
+// 2 - WARNING, something went wrong, but you can fix it with additional operations in code right now
+// 3 - INFO, it is interesting and useful for understanding a bug
+// 4 - DEBUG, more detaled info, dig deeper
+// 5 - TRACE, show algorithm flow
+// In real you may use own classification of debug levels.
+#define DBG(level, ...) \
```

Comment:
I understand this can be useful, but now in this PR is not a good time to add this macro.

For now please just use the existing `PWM_DBG` macro.

Reply:
I opened separate PR for DBG() macro #8050